tests/suites/controller/expect-scripts/juju-change-user-password.exp

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/expect -f
+#
+# This Expect script was generated by autoexpect on Tue Aug 30 14:45:41 2022
+# Expect and autoexpect were both written by Don Libes, NIST.
+#
+# Note that autoexpect does not guarantee a working script.  It
+# necessarily has to guess about certain things.  Two reasons a script
+# might fail are:
+#
+# 1) timing - A surprising number of programs (rn, ksh, zsh, telnet,
+# etc.) and devices discard or ignore keystrokes that arrive "too
+# quickly" after prompts.  If you find your new script hanging up at
+# one spot, try adding a short sleep just before the previous send.
+# Setting "force_conservative" to 1 (see below) makes Expect do this
+# automatically - pausing briefly before sending each character.  This
+# pacifies every program I know of.  The -c flag makes the script do
+# this in the first place.  The -C flag allows you to define a
+# character to toggle this mode off and on.
```

Comment:
I'm guessing we don't want to leave this commentary in all of our Expect scripts, though it *is* very good to have somewhere, especially if we have a reference script for people to start from when writing their expect scripts.